ui/app/src/modules/Content/Delete/DeleteDialog.tsx

```diff
@@ -235,11 +277,11 @@ export default function DeleteDialog(props: DeleteDialogProps) {
 function DeleteDialogWrapper(props: DeleteDialogProps) {
   const { items, onClose, onDismiss, onSuccess } = props;
   const [submissionComment, setSubmissionComment] = useState('');
+  const submissionCommentMaxLength = 250;
```

Comment:
Let's move to GlobaState.env and supply to all dialogs with submission comments

Reply:
done, other dialogs will be done in here https://github.com/craftercms/craftercms/issues/4305